documentation/2.0/content/userguide/tools/discover.md

```diff
@@ -77,21 +77,20 @@ The following environment variables may be set.
 
 Please provide the `STDOUT` and `STDERR` log streams in the GitHub Issue. If the summary is not listed (unhandled exception stack trace occurs), be sure and include the Oracle and WDT install versions and whether the tool was run in online or offline WLST mode. If possible, provide the model, variable and archive files, and the log file, `discoverDomain.log`, from location `<install home>\weblogic-deploy\log`.
 
-### Parameter table for discoverDomain
+### Parameter table for `discoverDomain`
 | Parameter | Definition | Default |
 | ---- | ---- | ---- |
 | `-archive_file` | The path to the archive file. |    |
-| `-admin_pass_env` | An alternative to entering the admin password at a prompt. The value is a ENVIRONMENT VARIABLE name that WDT will use to retrieve the password. |    |
+| `-admin_pass_env` | An alternative to entering the admin password at a prompt. The value is a environment variable name that WDT will use to retrieve the password. |    |
```

Comment:
*an environment - in a few places

Reply:
Oh my word, how embarrassing! Thank you @rakillen I was too late to fix it this time around, but I will definitely get to it soon!!!